compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.h

```diff
@@ -16,7 +16,7 @@ namespace mlir::iree_compiler::AMDAIE {
 
 /// Add passes to lower from MLIR-AIR through AIE. This is
 /// currently the default passes used for lowering after IREEs tiling.
-void addMLIRAIRAIELoweringPasses(OpPassManager &passManager);
+void addMLIRAIRAIELoweringPasses(OpPassManager &passManager, bool packPeel);
```

Comment:
Please add some comments about the new argument. Is this something temporary?

Reply:
Added comment to Passes.cpp. Yes eventually we would use the same pipeline as pad-pack. Please see comments inlined for the current pipeline differences.